package.json

```diff
@@ -4,6 +4,7 @@
   "workspaces": [
     "generators/*",
     "packages/*",
+    "packages/*/js",
```

Comment:
Do we need both `packages/*` and `packages/*/js`?

Reply:
Yeah, it's possible we can revisit this when we restructure packages for proper multi-language development. It's possible, for example, that `packages/**` would work.